src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -96,14 +123,23 @@ scope core
             octets extension;
         }
 
+        struct Reauthorize extends core::stream::Frame [0x00000005]
+        {
+            int64 authMask;
+            int64 authExpires;
```

Comment:
I think expiry need only be handled by the security nukleus and the WebSocket revalidate nukleus so this field is not needed.